lib/thrift/servers/binary_framed/acceptor_supervisor.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Thrift.Servers.BinaryFramed.AcceptorSupervisor do
+  @moduledoc """
+  A supervisor for acceptor processes.
+  """
+
+  alias Thrift.Servers.BinaryFramed.Acceptor
+  use Supervisor
+
+  def start_link(socket, server_module, handler_module, count) do
+    Supervisor.start_link(__MODULE__, [socket, server_module, handler_module, count])
+  end
+
+  def init([socket, server_module, handler_module, count]) do
+    children = for id <- (1..count) do
+      worker_id = "#{inspect handler_module}_acceptor_#{id}"
+      worker(Acceptor, [socket, server_module, handler_module], id: worker_id)
+    end
+    supervise(children, strategy: :one_for_one)
```

Comment:
I didn't use ranch because this British guy told me that it had some issues. 

One of the immediate issues I see with ranch is that it doesn't allow for `packet: 4`, but I suppose I could handle that myself.